### ISSUE TITLE
Problems adding new highlights

### ISSUE THREAD

**Issue by jorgegrivero** [unawe/spaceawe#64]
On the back-end of the website, on the section Highlights, when you add a new Highlight it goes at the end of the list instead of at the beginning, which makes that the most recent news don't appear first on the website frontpage. The main problem it seems there is no way to sort them out afterwards. Is there a way to fix this?

**Comment by slowe** [comment]
@jorgegrivero I've spent some time failing to work out why the site is displaying them in the order it is before I tried replicating what you'd said you were experiencing. I'm not sure how you managed to add an item to the end of the list. I've tried it on both my local development version of the site, the virtualbox version of the site, and the live version of the site. In each case a new highlight is added to the start of the list, not the end.

**Comment by slowe** [comment]
@jorgegrivero I've spent some time today failing to understand this. When I add a Highlight to any of the three versions of the website (local development version, Virtualbox version, the live website), it adds to the top (start) of the list in the back-end part of the site. I agree that the frontpage (in all three versions) is showing the list "backwards" (oldest highlight added is shown first) but I am unable to replicate the behaviour where the back-end adds to the end (bottom) of the list shown in the back-end. Hopefully I'm correct to assume that what is actually required is for the most recently added item to be shown first on the carousel on the front page with the oldest added item appearing last on the carousel. I've tweaked the front-end template to make it show the carousel in reverse order. Is this what was actually wanted?

Given what I might expect the most recent news to be on the front page, I guess that someone has (yesterday?) deleted then re-added all the front page news items as a temporary fix for this. If that is what has happened, they'll need to be put back in the order that now works (add from oldest to newest).
